ThirdPartyNoticeText.txt

```diff
@@ -85,12 +85,12 @@ DOM
 W3C License
 This work is being provided by the copyright holders under the following license.
 By obtaining and/or copying this work, you (the licensee) agree that you have read, understood, and will comply with the following terms and conditions.
-Permission to copy, modify, and distribute this work, with or without modification,�for any purpose and without fee or royalty is hereby granted, provided that you include the following 
+Permission to copy, modify, and distribute this work, with or without modification, for any purpose and without fee or royalty is hereby granted, provided that you include the following
 on ALL copies of the work or portions thereof, including modifications:
 * The full text of this NOTICE in a location viewable to users of the redistributed or derivative work.
 * Any pre-existing intellectual property disclaimers, notices, or terms and conditions. If none exist, the W3C Software and Document Short Notice should be included.
 * Notice of any changes or modifications, through a copyright statement on the new code or document such as "This software or document includes material copied from or derived 
-from [title and URI of the W3C document]. Copyright � [YEAR] W3C� (MIT, ERCIM, Keio, Beihang)." 
+from [title and URI of the W3C document]. Copyright (C) [YEAR] W3C(C) (MIT, ERCIM, Keio, Beihang)."

```

Comment:
That second one is an `(R)` (restricted trademark), not a `(C)` (copyright).

Reply:
Oh right, as I was replacing it for the third time I was not seeing it properly, thanks, for spotting it out 🔍

Reply:
What's the problem with these being Unicode symbols?